Gems/AWSCore/Code/Platform/Linux/AWSCoreEditor_Traits_Linux.h

```diff
@@ -7,4 +7,6 @@
  */
 #pragma once
 
-#define AWSCORE_EDITOR_RESOURCE_MAPPING_TOOL_ENABLED 0
+#define AWSCORE_EDITOR_RESOURCE_MAPPING_TOOL_ENABLED 1
+#define AWSCORE_EDITOR_PYTHON_DEBUG_ARGUMENT         ""
```

Comment:
we don't have debug version python on linux platform?

Reply:
the 'debug' argument was to switch to use the debug version of the python interpreter on windows. On linux, there is no separate release and debug versions, so the python.sh script does not handle that argument at all, resulting in python attempting to open a 'debug' file instead